test/quantization/test_quant_api.py

```diff
@@ -858,6 +859,57 @@ def test_int4wo_cuda_serialization(self):
             # load state_dict in cuda
             model.load_state_dict(sd, assign=True)
 
+    @unittest.skipIf(
+        "CPU" not in torch._C._dispatch_dump("torchao::qembeddingbag"),
+        reason="cpp kernels not built",
+    )
+    def test_embeddingbag_cpu(self):
```

Comment:
the test should be added here I think: https://github.com/pytorch/ao/blob/main/test/test_ops.py